injector/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'java-library'
+
+dependencies {
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'com.squareup:javapoet:1.11.1'
```

Comment:
Ty verze prosím do hlavního/společného gradle ;)

Reply:
su dost specificke (neviem kto by este isiel pouzivat `javapoet`), aj pri jednom pouziti ich chceme davat do hlavneho gradle filu? Pride mi, ze to ma zmysel az ked sa zacnu opakovat...

Reply:
To chápu, ale i tak bych to nenechával tu. Do budoucna až toho bude více a více, už dávno zapomeneme, že tu jsou někde definované verze a mohlo by to jen způsobovat problémy použitím různých verzí v jednom projektu. Už teď jsem viděl např, že je na dvou místech definovaný:
> javax.inject:javax.inject:1
tedy jestli dobře chápu, že to "1" je verze, mohli tam dám alespoň 1.0.0, tak jak je zvykem :)

Reply:
jj aj Dagger je `2.16` iba 🗡